Public/Src/App/Bxl/Args.cs

```diff
@@ -1014,6 +1019,11 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                             sign =>
                             sandboxConfiguration.UnsafeSandboxConfigurationMutable.MonitorFileAccesses = sign,
                             isUnsafe: true),
+                        OptionHandlerFactory.CreateBoolOption(
+                            "unsafe_OptimizedAstConversion",

```

Comment:
I through tyou needed it for BxlScriptAnalayzer app, just curious why doing this for bxl as well.

Reply:
There has been a thread that AST conversion takes a long time. While the binary fragment is not there yet, this can be a remedy. But the stuff implemented here is beneficial for constructing the binary fragment that we're doing currently.